ably/internal/ablyutil/msgpack.go

```diff
@@ -0,0 +1,38 @@
+package ablyutil
+
+import (
+	"bytes"
+	"io"
+
+	"github.com/ugorji/go/codec"
```

Comment:
Should be vendored

Reply:
No, not in this PR. I plan to make Godep directory go away by the time we integrate the integration branch to develop and use the new `go module` to manage dependencies.

But that work will be done later.

Reply:
Right, makes sense